math/src/components/Calculator.css

```diff
@@ -0,0 +1,55 @@
+.container {
+  top: 50%;
+  left: 50%;
+  right: 50%;
+  margin: 0 auto;
+  width: 300px;
+  text-align: center;
+  background-color: rgb(189, 182, 182);
+}
+
+.keys {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-rows: minmax(40px, auto);
+}
+
+.zero {
+  grid-column: 1 / 3;
+}
+
+.keys button {
+  outline: none;
+  border: 1px solid rgb(189, 182, 182);
+}
+
+.operator {
+  background-color: rgb(255, 136, 0);
+}
+
+.display {
+  height: 45px;
+  margin-bottom: 0;
+  line-height: 45px;
+  color: #fff;
+  display: inline-block;
+  width: 100%;
+  overflow: hidden;
+  background-color: rgb(189, 182, 182);
+  border: none;
+  outline: none;
+  text-align: right;
+  font-size: 20px;
+  font-weight: 500;
+}
+
+input::-webkit-outer-spin-button,
+input::-webkit-inner-spin-button {
+  /* display: none; <- Crashes Chrome on hover */
```

Comment:
- [ ] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.

Reply:
done